tests/end-to-end/external-solution-integration/resourceskit/k8s_resources_client.go

```diff
@@ -0,0 +1,96 @@
+package resourceskit
+
+import (
+	"github.com/kyma-project/kyma/components/application-operator/pkg/client/clientset/versioned"
+	model "k8s.io/api/apps/v1"
+	core "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+)
+
+type K8sResourcesClient interface {
```

Comment:
I see `k8sResourcesClient` as redundant. Dependants could you k8s interfaces directly. For example for [Services](https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/service.go#L39) or [Deployments](https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/apps/v1/deployment.go#L40).